main_test.go

```diff
@@ -34,7 +36,9 @@ func (f *ComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
 	ctx.BeforeScenario(func(*godog.Scenario) {
 		apiFeature.Reset()
 		datasetFeature.Reset()
-		f.MongoFeature.Reset()
+		if err := f.MongoFeature.Reset(); err != nil {
+			log.Error(context.Background(), "failed to reset mongo feature", err)
```

Comment:
Can the ctx from line 36 be used here ?

Reply:
I tried to use it but it's not possible (it's a different type of context, so it can't be used by the log)